Make the doc test for StartLiveData-v1 more stable

**Describe the bug**
The doc test for StartLiveData-v1 regularly hits a Seg fault, especially on the nightly pipelines. 

**To Reproduce**
Console output

```
Document: algorithms/StartLiveData-v1
-------------------------------------
FakeISISEventDAE-[Notice] FakeISISEventDAE started
StartLiveData-[Notice] StartLiveData started
ISISLiveEventDataListener-[Notice] Number of periods 1
ISISLiveEventDataListener-[Notice] Number of spectra 100
ISISLiveEventDataListener-[Notice] Loading instrument MUSR ... 
ISISLiveEventDataListener-[Notice] Instrument loaded.
LoadLiveData-[Notice] Performing chunk processing using Rebin
LoadLiveData-[Notice] Performing the Replace operation.
StartLiveData-[Notice] StartLiveData successful, Duration 0.01 seconds
MonitorLiveData-[Notice] MonitorLiveData started
MonitorLiveData-[Notice] Loading live data chunk 0 at 00:04:51
MonitorLiveData-[Warning] MonitorLiveData: Execution cancelled by user.
FakeISISHistoDAE-[Critical] FakeISISHistoDAE: Execution cancelled by user.
FakeISISEventDAE-[Warning] FakeISISEventDAE: Execution cancelled by user.
Exception in thread Thread-2 (startFakeDAE):
Traceback (most recent call last):
  File "/jenkins_workdir/workspace/main_nightly_deployment/source/mambaforge/envs/mantid-developer/lib/python3.10/threading.py", line 1016, in _bootstrap_inner
    self.run()
  File "/jenkins_workdir/workspace/main_nightly_deployment/source/mambaforge/envs/mantid-developer/lib/python3.10/threading.py", line 953, in run
    self._target(*self._args, **self._kwargs)
  File "<doctest exStartLiveDataEvent[0]>", line 9, in startFakeDAE
  File "/jenkins_workdir/workspace/main_nightly_deployment/source/Framework/PythonInterface/mantid/simpleapi.py", line 1082, in __call__
    algm.execute()
KeyboardInterrupt
Segmentation fault
ninja: build stopped: subcommand failed.

```


**Expected behavior**
The doc test should not seg fault as often as it does

**Platform/Version (please complete the following information):**
 - OS: Linux

**Additional context**
As this is making the nightly pipelines unstable this could have an adverse impact on release. Therefore it is being marked as a high priority to resolve.

--- a/docs/source/algorithms/StartLiveData-v1.rst
+++ b/docs/source/algorithms/StartLiveData-v1.rst
@@ -151,15 +151,15 @@ Usage
 
     oldFacility = ConfigService.getFacility().name()
     thread = Thread(target = startFakeDAE)
-    thread.start()
-    time.sleep(2) # give it a small amount of time to get ready
-    if not thread.is_alive():
-        raise RuntimeError("Unable to start FakeDAE")
 
     try:
+        thread.start()
+        time.sleep(2) # give it a small amount of time to get ready
+        if not thread.is_alive():
+            raise RuntimeError("Unable to start FakeDAE")
         captureLive()
-    except Exception:
-        print("Error occurred starting live data")
+    except Exception as e:
+        print(f"Error occurred starting live data: {str(e)}")
     finally:
         thread.join() # this must get hit
 
@@ -216,15 +216,15 @@ Output:
 
     oldFacility = ConfigService.getFacility().name()
     thread = Thread(target = startFakeDAE)
-    thread.start()
-    time.sleep(2) # give it a small amount of time to get ready
-    if not thread.is_alive():
-        raise RuntimeError("Unable to start FakeDAE")
 
     try:
+        thread.start()
+        time.sleep(2) # give it a small amount of time to get ready
+        if not thread.is_alive():
+            raise RuntimeError("Unable to start FakeDAE")
         captureLive()
-    except Exception:
-        print("Error occurred starting live data")
+    except Exception as e:
+        print(f"Error occurred starting live data: {str(e)}")
     finally:
         thread.join() # this must get hit
 

